src/webdriver.rs

```diff
@@ -158,7 +158,7 @@ impl WebDriver {
 // Contains Element Handling
 impl WebDriver {
     /// Requests an elements from the webpage, given the specified selector and query string
-    pub fn query_element(&self, selector: Selector, query: &str) -> reqwest::Result<Element> {
+    pub fn find_element(&self, selector: Selector, query: &str) -> reqwest::Result<Element> {
```

Comment:
Could you leave the query_element functions as is, and just add the find_element function instead, as well as adding the <code>#[deprecated]</code> attribute to query_element?

Reply:
ok, I will add back `query_`* methods and have then delegate to `find_`* methods and add the attribute.

Reply:
Just want to verify this is how I should use deprecated. 

- Unsure what version to use in since. I expect the deprecation will not actually happen until the next version you publish.
- Should I change the verbiage of  `query_element` to be different from `find_element`. 
```
impl WebDriver {
    /// Requests an elements from the webpage, given the specified selector and query string
    #[deprecated(since = "0.1.1" note = "query_element did not follow WebDriver naming convention, use find_element")]
    pub fn query_element(&self, selector: Selector, query: &str) -> reqwest::Result<Element> {
        find_element(self, selector, query)
    }

    /// Requests an elements from the webpage, given the specified selector and query string
    pub fn find_element(&self, selector: Selector, query: &str) -> reqwest::Result<Element> {
    ...
    }
...
}
```

Reply:
You can leave it as <code>0.1.2</code>, and I'll bump the version number on crates.io, so that way people still on the old version won't see the deprecation warning. And just change the tense to present, so that the description reads <code>query_element does not follow WebDriver naming convention, use find_element.</code>

Reply:
Made the requested changes. Added tests for both the `find_` and `query_` versions.